### PR TITLE
Add debug noclip

### PR DESCRIPTION
**Pull request by Ephememory**

### CODE REVIEW

**Review by matekdev — APPROVED** [review]
This is a nice to have while we develop, can be purged whenever we are done. What do you think? @kolexxx

File: code/Player/Movement/WalkController.Noclip.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
+using Sandbox;
+
+namespace TTT;
+
+public partial class WalkController
+{
+#if DEBUG
```
Comment (kolexxx) [inline code/Player/Movement/WalkController.Noclip.cs:7]: Should this be outside of the class?

**Comment by kolexxx** [comment]
> This is a nice to have while we develop, can be purged whenever we are done. What do you think? @kolexxx

As long as it's useful for debugging, we can always have it as long as it's under that preprocessor definition.

**Review by kolexxx — APPROVED** [review]

File: code/Player/Movement/WalkController.Noclip.cs (hunk 1 above)
Comment (Ephememory) [reply code/Player/Movement/WalkController.Noclip.cs]: Ah yeah, you're right. Fixed

**Comment by kolexxx** [comment]
Thank you for contributing

**Pull request merged**